packages/kubernetes-1.23/kubernetes-1.23.spec

```diff
@@ -103,6 +103,35 @@ Conflicts: (%{_cross_os}image-feature(no-fips) or %{_cross_os}kubelet-1.23-bin)
 %description -n %{_cross_os}kubelet-1.23-fips-bin
 %{summary}.
 
+%package -n %{_cross_os}kube-proxy-1.23
+Summary: Container cluster node proxy
+Requires: %{_cross_os}conntrack-tools
+Requires: %{_cross_os}containerd
+Requires: %{_cross_os}ecr-credential-provider-1.23
```

Comment:
What provides these versioned `ecr-credential-provider` packages?

Reply:
Wishful thinking :) I didn't test the 1.23 AMI build.